File: ContactApp/app/src/main/java/com/example/contactapp/MainActivity.java
package com.example.contactapp;

import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);
	}

	@Override
	public boolean onCreateOptionsMenu(Menu menu) {
		MenuInflater inflater = new MenuInflater(this);
		inflater.inflate(R.menu.actionmenu, menu);
		return super.onCreateOptionsMenu(menu);
	}

	@Override
	public boolean onOptionsItemSelected(@NonNull MenuItem item) {
		switch (item.getItemId()) {
			case R.id.mnuSortName:
				break;
			case R.id.mnuSortPhone:
				break;
			default:
				break;
		}
		return super.onOptionsItemSelected(item);
	}
}

File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Toast
import retrofit2.Call
import retrofit2.Response
import javax.security.auth.callback.Callback

class MainActivity : AppCompatActivity() {

	lateinit var users: MutableList<User>
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_main)

		val result = UserApi.getInstance().getAllImage()
		result.enqueue(object : Callback, retrofit2.Callback<List<User>> {
			override fun onResponse(call: Call<List<User>>, response: Response<List<User>>) {
				Log.e(this.javaClass.simpleName, response.body().toString())
				users = response.body() as MutableList<User>



			}

			override fun onFailure(call: Call<List<User>>, t: Throwable) {
				Log.e(this.javaClass.simpleName, t.message.toString())
			}

		})
	}
}